daemon/src/comm/dbus.rs

```diff
@@ -107,6 +107,7 @@ pub(crate) fn validate_device_handle(device_handle: &str) -> Result<(), FpgadErr
 /// # Examples
 ///
 /// ```
+/// use std::path::Path;
```

Comment:
linter complains about the example not having imports otherwise

Reply:
I believe you can use `#` to prevent this weird imports appear in rust docs
```suggestion
/// #use std::path::Path;
```